### ISSUE TITLE
The results of yolo_v3 detector are different.

### ISSUE THREAD

**Issue by DeHysolate** [AlexeyAB/darknet#602]
I used the same cfg, weight and same image, but the results of running pjreddie / darknet in a Linux environment and AlexeyAB / darknet in a Windows environment are different.

I wonder what causes this result.

This is results. (using same command: 'darknet detector test ~~~')

pjreddie / darknet  in Linux
![image](https://user-images.githubusercontent.com/16909931/38476038-b4a03e74-3be6-11e8-8161-f556b501e9bb.png)

AlexeyAB / darknet in a Window
![image](https://user-images.githubusercontent.com/16909931/38476150-30e5f8fc-3be7-11e8-9071-b31c3f62bb07.png)

**Comment by XP-CHRC** [comment]
In ubuntu16.04, same cfg and weights, the AlexeyAB/darknet is better than the pjreddie/darknet, maybe you can check the function test_detector under the file detector.c, there are some differences.

**Comment by DeHysolate** [comment]
@XP-CHRC 
I compared the original with AlexeyAB's, but found no significant change.
When the original and AlexeyAB's are using the same cfg and weight, does the result differ?

**Comment by DeHysolate** [comment]
@AlexeyAB 

- Can you show whole command that you used?
--detector test coco.data yolov3_train.cfg yolov3_train.weights save.jpg -i 0 -thresh 0.25
- Check that you didn't specify -thresh in the command.
-- -thresh 0.25
- Did you use default coco-model yolov3.weights or trained by yourself?
-- myself based on yolov3.weight
- Can you drag-n-drop this original image (without bound boxes) to next message here? I'll try to detect.
-- Now I'm out and I can not capture it.

**Comment by AlexeyAB** [comment]
@DeHysolate Can you get different results on original / fork repositories using the default `yolov3.weights` model?

**Comment by DeHysolate** [comment]
@AlexeyAB I'll try it tomorrow and upload image because I'm out now. Thanks for your kindness!!

**Comment by DeHysolate** [comment]
@AlexeyAB 
Result of using default weight (coco-model yolov3.weight)
The bounding box results are slightly different.
The probability results are also different. (origianl:100%,99% / fork:100%,97%)

original repositories
![image](https://user-images.githubusercontent.com/16909931/38530047-5d3910c6-3ca3-11e8-8d94-4196c8baa7a5.png)

fork repositories
![image](https://user-images.githubusercontent.com/16909931/38530034-46240ad0-3ca3-11e8-8178-be4f90b389fb.png)

This is origianl image.
![save_o](https://user-images.githubusercontent.com/16909931/38530130-d27c35e8-3ca3-11e8-9578-d3382ca94941.jpg)

**Comment by AlexeyAB** [comment]
@DeHysolate 
Yes, there is a little difference due to different resize approaches: https://github.com/AlexeyAB/darknet/issues/232#issuecomment-336955485
